### ISSUE TITLE
Reduce the AP cost of some pistol perks

### ISSUE THREAD

**Issue by Anxoczt** [long-war-2/lwotc#790]
Some of these perks are way too pricy, it wouldn't be such a problem if they wouldn't compete with other XCOM skills who, at similar rank can do way more.

Lets see
Return fire and Quickdraw sits at 5 AP a pop, this seems alright. Return fire is trash tho because firing, with reaction penalty, against enemy targets in cover is not going to happen. Quickdraw is a good perk.

Clutchshot is alright for 15 AP since its a nondodgeable attack, perfect for finishing wounded targets.

Gunslinger costs 15 AP, aside from pistols questionable accuracy i feel what it can potentially do, this is fair enough price.

Lightning hands should be reduced to 10 AP, this perk is good... in vanilla WOTC, but not that great in LW2 because everyone is tougher and there is just more of them. Its not a utter trash like return fire though.

Fanfire should be cheaper, 20 if not even less. Since the redscreen and hack boost stacking nerf, this ability got battered considerably and its only saving grace is bluescreen rounds vs robots so the pistol user's DPS can actually get close to other classes main weapons... for one turn before it goes on cooldown.

Faceoff should be cheaper at 20 if not less because pistols are not that powerful in LWOTC.

All of these problems werent present in LW2 because you could just whip them to shape in ubermench chamber with just costing some time off for them before you could use them. But here you have to spend AP and these are competing with other, XCOM perks which can vary from free grenades, bonus defences to increased damage output.

TLDR - Pistol perks should be cheaper because you have to actually buy them now over XCOM skills. Wasnt problem in vanilla since you could train them for free in advanced combat warfare centre.

**Comment by Grobobobo** [comment]
Not nescessary with xcom's increased soldier ap gain.
